src/main/java/com/digirati/taxonomy/manager/lookup/normalisation/TextNormaliser.java

```diff
@@ -57,24 +60,48 @@ private void initialisePipeline(String languageKey, String languageName) {
     }
 
     /**
-     * Applies the NLP pipeline to a piece of input text to produce a normalised version of it. This
-     * includes tokenisation, sentence splitting, part of speech tagging, and lemmatisation (if this
-     * is supported for the language in question).
+     * Applies the NLP pipeline to a piece of input text to produce a list of the {@link
+     * Word}s it contains (i.e. all words, excluding stopwords). The NLP pipeline includes
+     * tokenisation, sentence splitting, part of speech tagging, and lemmatisation (if this is
+     * supported for the language in question).
      *
-     * @param text a non-normalised piece of text
-     * @return the normalised form of the input text
+     * @param text a piece of text from which to extract the content words
+     * @return the list of content words contained in the input text
      */
-    public String normalise(String text) {
-        StringBuilder normalisedText = new StringBuilder();
+    public List<Word> extractContentWords(String text) {
         CoreDocument document = new CoreDocument(text);
         pipeline.annotate(document);
-        document.tokens().stream()
+        return document.tokens().stream()
                 .filter(this::isContentWord)
-                .forEach(coreLabel -> normalisedText.append(coreLabel.lemma()).append(" "));
-        return normalisedText.toString().trim();
+                .map(Word::new)
+                .collect(Collectors.toList());
     }
 
     private boolean isContentWord(CoreLabel label) {
         return !stopwords.contains(label.originalText()) && !stopwords.contains(label.lemma());
     }
+
+    /**
+     * Converts a list of {@link Word}s into a single normalised string.
+     *
+     * @param contentWords the {@link Word}s to convert
+     * @return a single normalised string built from the input {@link Word}s
+     */
+    public String normalise(List<Word> contentWords) {
+        StringBuilder normalisedText = new StringBuilder();
+        contentWords.forEach(coreLabel -> normalisedText.append(coreLabel.getLemma()).append(" "));
+        return normalisedText.toString().trim();
+    }
+
+    /**
+     * Applies the NLP pipeline to a piece of input text to produce a normalised version of it. This
+     * includes tokenisation, sentence splitting, part of speech tagging, and lemmatisation (if this
+     * is supported for the language in question).
+     *
+     * @param text a non-normalised piece of text
+     * @return the normalised form of the input text
+     */
+    public String normalise(String text) {
```

Comment:
This method only appears to be used by tests - can we get rid of it, or am I missing something?

Reply:
I've left this in in this PR as a convenience method for tests, but we're going to need it when we get round to normalising the terms going into the Aho-Corasick algorithm

Reply:
> I've left this in in this PR as a convenience method for tests
We shouldn't be writing production code that only exists to support test code.


> we're going to need it when we get round to normalising the terms going into the Aho-Corasick algorithm

We can add it when we do that.